frontend/tsconfig.eslint.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": ".",
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "isolatedModules": true,
+    "jsx": "react",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "es5"
+  },
+  "include": ["src"]
+}
```

Comment:
eslint lints more files than the typechecker checks, so this is a different file. Once we move the type checker to work with test files, we can remove the linter file as well.

Reply:
Can you add this note in a comment in the file? (Also, what do you mean by "so this is a different file"? Different from what?) I think json doesn't allow comments, so maybe something like:

```
{
  "__TODO": "Remove this once we start typechecking test files. eslint lints more files than the typechecker checks, ..."
  "compilerOptions": ...
}
```